### ISSUE TITLE
Include issue template

### ISSUE THREAD

**Issue by anaschwendler** [datasciencebr/rosie#112]
Rosie already have its own `PULL_REQUEST_TEMPLATE.md` but now we can add the issue template to the project.

The text:
```markdown
This template is here to help you to write your issue ;)

**What is the problem?**
__Try to give as many details about the bug/new feature/enhancement as possible, but keep it friendly for new comers__

**How can this be addressed?**
__New comers might need help on how to approach or build a solution, maybe a step-by-step on how to address the issue can help__

**Who could help with this issue?**
__We know sometimes a good session of pair programming can do wonders, maybe indicate someone that can be reached to help out and discuss ideas. If you don't know someone but is up for a pair programming session use this space to tell us o/__

**Labels**
__Feel free to write up labels that you think might apply here. e.g. [first-timers-only]  [bug] __
```

**How can this be addressed?**
By adding a `ISSUE_TEMPLATE.md` file in the `.github` folder.

**Comment by Securter** [comment]
Hi @anaschwendler I've submitted a PR regarding this issue, if there is anything else needed to be added or changed I'm here to help 👍

**Comment by cuducos** [comment]
Closed by #116
